dd-sdk-android/src/main/json/trace/span-schema.json

```diff
@@ -178,7 +183,8 @@
                 }
               },
               "required": [
-                "connectivity"
+                "connectivity",
+                "sim_carrier"
```

Comment:
sim_carrier should not be required I think (e.g. when you're using wifi on a device without SIM card)